server/src/main/java/org/opensearch/index/remote/RemoteRefreshSegmentTracker.java

```diff
@@ -555,6 +594,8 @@ public Stats(StreamInput in) throws IOException {
         @Override
         public void writeTo(StreamOutput out) throws IOException {
             shardId.writeTo(out);
+            out.writeLong(localRefreshClockTimeMs);
+            out.writeLong(remoteRefreshClockTimeMs);
```

Comment:
Agree, bcw issues can only arise if different code is running on different nodes in the cluster.  but since api is still experimental i am keeping the related fields together.